### PR TITLE
[Merged by Bors] - api: node info

### PR DESCRIPTION
**Pull request by dshulyak**
requires: https://github.com/spacemeshos/go-spacemesh/pull/4662
api: https://github.com/spacemeshos/api/pull/250

add handler with:
- network hrp, as it is hardcoded in smapp
- first/effective genesis, for clients to understand that checkpoint happened
- epoch size, just useful info

### CODE REVIEW

**Comment by dshulyak** [comment]
@brusherru @kacpersaw

File: config/config.go [hunk 1]
```diff
@@ -111,6 +109,8 @@ type BaseConfig struct {
 
 	DatabaseConnections     int  `mapstructure:"db-connections"`
 	DatabaseLatencyMetering bool `mapstructure:"db-latency-metering"`
+
+	NetworkHRP string `mapstructure:"network-hrp"`
```
Comment (fasmat) [inline config/config.go]: Do we need the global if the HRP is part of the config? If it is too much effort, I would at least create an issue for this to refactor in the future.

Comment (dshulyak) [reply config/config.go]: the main problem is that hrp is used when address is printed. there are a couple of logs with addresses across the codebase. maybe we can reconsider

**Review by fasmat — APPROVED** [review]

**Comment by dshulyak** [comment]
bors merge

**Pull request closed without merging**